exopy_hqc_legacy/instruments/drivers/dll/sp_adq14.py

```diff
@@ -99,6 +104,7 @@ def open_connection(self):
 
         """
         cu = ADQControlUnit(self._dll)
+        cu.enable_logging(3, b"C:\\Logs")
```

Comment:
I am not a big fan of this hard-coded. Could we do something like at least trying to write in the user folder with a more scpecific folder name ?